.github/workflows/test.yaml

```diff
@@ -37,6 +37,8 @@ jobs:
           python -m pip install -U https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-2.0.0.dev0-cp37-cp37m-manylinux2014_x86_64.whl
           if [ -f requirements-test.txt ]; then python -m pip install -r requirements-test.txt; fi
           HOROVOD_WITH_GLOO=1 HOROVOD_WITHOUT_MPI=1 HOROVOD_WITHOUT_MXNET=1 pip install git+https://github.com/horovod/horovod.git
+          # Install PTL master
+          pip install -U https://github.com/PyTorchLightning/pytorch-lightning/archive/master.zip
```

Comment:
when will we be able to pin?

Reply:
This can actually be removed now. The necessary change has been included in PTL 1.2.4 release